core/chains/evm/txmgr/evm_tx_store.go

```diff
@@ -1931,6 +1912,38 @@ AND evm_chain_id = $2`, timeThreshold, chainID.String())
 	if err != nil {
 		return pkgerrors.Wrap(err, "TxmReaper#reapEthTxes batch delete of fatally errored evm.txes failed")
 	}
+	// Delete old 'confirmed' evm.txes that were never finalized
```

Comment:
There is not reason to split the query into two. Instead use:
`AND evm.txes.state = 'confirmed' OR evm.txes.state = 'finalized'`

Reply:
That's right. The reason this was kept separate was so we can log an error when purging confirmed transactions. I agree though about the log level. I lowered a similar log in the Finalizer from Crit to Error but missed this one